src/makeInfinityAdder.js

```diff
@@ -5,6 +5,30 @@
  */
 function makeInfinityAdder() {
```

Comment:
The outer function `makeInfinityAdder` should return a function that initiates the accumulator. The current implementation has a flaw where the total is reset every time a new value is added, which is not the desired functionality.